### DISCUSSION TITLE
Replace the default cmd command

### DISCUSSION THREAD

**Discussion by ZengGengSen** [chrisant996/clink#598] [Ideas]
There is a mkdir default in cmd, but I want to replace the mkdir directive with another software (such as uutils/coreutils). Does clink provide this, or how should it work, to achieve the same effect

**Comment by chrisant996** [comment]
It isn't a Clink thing, it's a Windows thing.

Search for documentation on `doskey`.

**Reply by ZengGengSen** [reply]
Thanks
